src/components/tide-chart.tsx

```diff
@@ -144,6 +144,19 @@ export function TideChart() {
                   tickLine={false}
                   axisLine={false}
                 />
+                {/* TODO: add these reference areas for sunrise/sunset */}
```

Comment:
Surfline does this, would be cool to add it in since the sunlight API gives this data to us "for free" in the response body anyway.